.github/workflows/CI.yml

```diff
@@ -7,6 +7,8 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Out of interest, what does a cron schedule do?

Reply:
Runs the build and tests every day at midnight (to check that any updates to dependencies haven't broken anything if we go through periods without PRs). Cron is an (old) unix standard for doing this which GitHub actions have chosen, nowadays modern systems do this with systemd rather than cron.